File: services/device/src/main/java/com/example/iter/device/dto/response/PresignedImageUploadItemResponse.java
package com.example.iter.device.dto.response;

import com.example.iter.common.image.CaptureView;

import java.time.LocalDateTime;
import java.util.Map;

public record PresignedImageUploadItemResponse(
        CaptureView captureView,
        String objectKey,
        String uploadUrl,
        Map<String, String> requiredHeaders,
        LocalDateTime expiresAt
) {
}

File: services/device/src/main/kotlin/iter/device/dto/response/PresignedImageUploadItemResponse.kt
package iter.device.dto.response

import iter.common.image.CaptureView
import java.time.LocalDateTime

data class PresignedImageUploadItemResponse(
    val captureView: CaptureView?,
    val objectKey: String,
    val uploadUrl: String,
    val requiredHeaders: Map<String, String>,
    val expiresAt: LocalDateTime
)
